loadtest/config.go

```diff
@@ -95,6 +95,8 @@ type InstanceConfiguration struct {
 	PercentReplies float64 `default:"0.5" validate:"range:[0,1]"`
 	// The percentage of replies that should be in long threads
 	PercentRepliesInLongThreads float64 `default:"0.05" validate:"range:[0,1]"`
+	// The percentage of post that are marked as urgent
+	PercentUrgentPosts float64 `default:"0.05" validate:"range:[0,1]"`
```

Comment:
Why is this different from 0.001?

Reply:
oh, I thought I change all occurrences, thanks Agniva will change